storybookjs/storybook
# Inline emotion css calls that require theme to avoid using state

Issue:

`getDerivedStateFromProps` can be called before the prop `theme` is set by react-emotion which causes errors.

## What I did

Inlined the `css` calls instead of placing them on the component's state. `withTheme` will rerender this component when the `theme` changes so there is no need to use `getDerivedStateFromProps`.

## How to test

Is this testable with Jest or Chromatic screenshots?
Does this need a new example in the kitchen sink apps?
Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

fixes #3840

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
